charter.md

```diff
@@ -53,15 +53,207 @@ The following responsibilities belong to the Steering Committe, but may be deleg
+
+COMMUNITY CONSIDERATIONS
+
+No Confidence
+
+- [ ] Provide guidelines on "no confidence"
+  - Process transparency could be considered inversely proportional to the importance of these guidelines
+
+Removal
+
+- [ ] Scope of removal of members (one, many)
+- [ ] Precedence for removal scenarios
+- [ ] What details of removal scenarios should be public/private?
+  - Call for removal?
+  - Reason for removal?
+  - Vote on removal?
+  - What are the public artifacts of the removal process?
+- [ ] What details of election results should be public/private?
+- [ ] Consequences of removal scenarios
+  - Are individuals ineligible for re-election?
+    - Should community members have an opinion in this decision?
+- [ ] When is removal appropriate?
+- [ ] How should CoC concerns be considered in the removal process? Should they be?
+- [ ] Guard against "popularity contests" or "burning at the person" in committee voting scenarios
+- [ ] Are there antitrust implications in the removal process?
+- [ ] Special elections may not have representative turnout
+  - Define guidance on acts and thresholds to consider in a vote for removal
+- [ ] Is removal an extraordinary power to grant?
+  - "Policies like this have to work for the good of the community both when used well and when used poorly."
+- [ ] Define removal process guidelines
+  - "brief rationale for removal" + short time + "brief rebuttal"
+  - Offer resignation as an option
+
+-->
+
+### No confidence
+
+In the event a Steering Committee member receives a vote of no confidence from
+two-thirds of the remaining Steering Committee members, that person must either
+resign or call a special election in which the project community will vote on
+whether that person remains or is removed from the Steering Committee. A
+majority of eligible voters participating in the election must vote in favor of
+the individual remaining; otherwise, they will be removed. The individual's
+participation in the Steering Committee will be suspended during the pendency
+of the special election.
+
+## Voting
+
+<!--
+
+COMMUNITY CONSIDERATIONS
+
+Voting
+
+- [ ] Better define voting requirements in specific scenarios
+  - Consider 5/7, as 4/6 is a simple majority of the entire committee
+  - Clarify requirements respective to size of committee and round up e.g., 7 * 2/3 = 4.6666..., which would be 5 votes out of 7
+  - Are there scenarios where unanimity should be considered?
+  - [Stephen] What about a combination of both?
+
+-->
+
+In the course of the committee's operations, members will be expected to vote
+on decisions within the body's purview.
+
+These votes may proceed on agreed-upon platforms by the committee, such as:
+
+* a pull request
+* an issue
+* a Steering Committee [meeting](#meetings)
+* a mailing list
+
+Unless otherwise specified by a process, the requirement for passing a vote is
+a **_two-thirds supermajority of the
```

Comment:
I think this is refactoring rather than defining for the first time, or I would complain that this is an excessively high bar and > 50% seems more natural for routine business.

Reply:
@parispittman @justaugustus - need to address this?

Reply:
I don't think you need to fix this in this PR.

Reply:
Yeah, this is a change to current process, and usually a simple majority has been fine.